CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.8.4
+
+<!-- generated comparing v5.8.3..master -->
+
+_Jun 13, 2022_
+
+A big thanks to the 22 contributors who made this release possible.
+This release is mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.8.4`
+
+- &#8203;<!-- 37 -->[Autocomplete] Add `getOptionKey` prop (#32910) @pushys
```

Comment:
I think that we should revert, I left a comment. I believe that we have a better solution.

```suggestion
```